ccd-definition/CaseField.json

```diff
@@ -603,5 +603,22 @@
     "Label": "Upload C2",
     "FieldType": "C2DocumentBundle",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "c21Orders",
+    "Label": "C21 Order",
+    "FieldType": "Collection",
+    "FieldTypeParameter": "C21Order",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "c21Order",
+    "Label": "Create an order",
+    "FieldType": "C21Order",
+    "SecurityClassification": "Public"
```

Comment:
add "Comment": "the reason why there is a c21Order and collection" here

Reply:
Resolved but not actioned yet, worry it will be forgotten!

Reply:
Forgot to do this - adding it in now 👍